elements/lisk-chain/src/block_assets.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { codec } from '@liskhq/lisk-codec';
+import { blockAssetSchema } from './schema';
+
+export interface BlockAsset {
+	moduleID: number;
+	data: Buffer;
+}
+
+export class BlockAssets {
```

Comment:
It's more of a question, can we drop "block" and just keep it as `Assets` ? 
thinking to align naming with other block property `payload` & `Transaction`